tests/test_anim.py

```diff
@@ -164,103 +110,53 @@ def test_logging_anim(
     output = capsys.readouterr().out
     assert output.endswith("Run identifier 1\n")
 
-    private_cli.log_anim(
-        database=tmp_db,
-        run_id=1,
-        query_fasta=input_genomes_tiny / "MGV-GENOME-0264574.fas",
-        subject_fasta=input_genomes_tiny / "MGV-GENOME-0266457.fna",
-        deltafilter=input_genomes_tiny
-        / "intermediates/ANIm/MGV-GENOME-0264574_vs_MGV-GENOME-0266457.filter",
-    )
-
-    # Check the recorded comparison values
     session = db_orm.connect_to_db(tmp_db)
-    assert session.query(db_orm.Comparison).count() == 1
-    comp = session.query(db_orm.Comparison).one()
-    query = "689d3fd6881db36b5e08329cf23cecdd"  # MGV-GENOME-0264574.fas
-    subject = "78975d5144a1cd12e98898d573cf6536"  # MGV-GENOME-0266457.fna
-    pytest.approx(
-        comp.identity,
-        get_matrix_entry(
-            input_genomes_tiny / "matrices" / "ANIm_identity.tsv", query, subject
-        ),
-    )
-    pytest.approx(
-        comp.aln_length,
-        get_matrix_entry(
-            input_genomes_tiny / "matrices" / "ANIm_aln_lengths.tsv", query, subject
-        ),
-    )
-    pytest.approx(
-        comp.sim_errors,
-        get_matrix_entry(
-            input_genomes_tiny / "matrices" / "ANIm_sim_errors.tsv", query, subject
-        ),
-    )
-    pytest.approx(
-        comp.cov_query,
-        get_matrix_entry(
-            input_genomes_tiny / "matrices" / "ANIm_coverage.tsv", query, subject
-        ),
+    run = session.query(db_orm.Run).one()
+    assert run.run_id == 1
+    hash_to_filename = {_.genome_hash: _.fasta_filename for _ in run.fasta_hashes}
+
+    subject_hash = list(hash_to_filename)[1]
+    private_cli.anim(
+        tmp_dir,
+        session,
+        run,
+        input_genomes_tiny,
+        hash_to_filename,
+        {},  # not used for ANIm
+        query_hashes=set(hash_to_filename),  # order should not matter!
+        subject_hash=subject_hash,
     )
-    session.close()
-    tmp_db.unlink()
-
-
-def test_logging_anim_bad_alignment(
```

Comment:
I dropped this entire new test, which is a duplicate of the 3 viruses but for the bad alignment set.

I don't think we need here this given the same example is run in full via the snakemake ANIm tests.

Reply:
That's fair. I can see now that there is no point in wasting time regenerating the same files over and over if the part of the code that captures `None` when there are no alignments is already tested elsewhere.